doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,605 @@
+            InterfaceID uint16
+        }
+
+    Maps a connection pair consisting of two ConnectionPoints to a list of policy indices.
+    This map shows for each connection pair which policy indices are supported, which can be one or multiple policies.
+    A ConnectionPoint is either an interface, an IP range or wildcard.
+    For all intermediary hops interface to interface connection points will be used whereas interface to IP range is used for the last hop.
+- IndexIdentifierMap
+    A policy index is to be embedded in the HBH extension and therefore has to be minimal in size.
+    The size of a policy index is 8 bits, whereas identifiers can be a multiple of this (especially global identifiers).
+    The policy index is thus different to the policy identifier. In order to decode which policies are supported on which interfaces,
+    a mapping is required from policy index to local and global identifiers.
+    This mapping is provided by this map.
+- IdentifierDescriptionMap
+    Global identifiers can be found in a global datastore, but local identifiers are specific to an AS.
+    This map maps a local policy identifier to its corresponding description.
+- MPLSMaps
+    Routers need to be aware of the supported policy indices and the corresponding MPLS config they need to apply to packets to
+    enforce the policy in the internal network.
+    Routers periodically fetch this map from the control service.
+    A hash of the MPLS map is maintained, such that routers only have to update if their hash differs from the one at the control service.
+- RemotePolicyCache
+    When a local policy is queried at a remote AS, the resulting policy description is cached at the requesting AS' FABRID Manager,
+    such that subsequent requests can be served from cache.
+
+
+PCB dissemination
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The *IndexIdentifierMap* and *SupportedIndicesMap* are included in a (unsigned) detachable extension in the PCBs for an AS.
+Hashes of these maps are maintained in a Signed AS Entry, such that the authenticity of these maps can be verified.
+The detachable extension can also be present in the PCB, i.e. it does not have to be detached in all cases, e.g. if there are only very few policies.
+If the maps are detached, they can be fetched from the control service of that AS and the received maps can be verified with the hashes.
+To ensure a consistent hash calculation, the key entries of these maps have to be sorted, such that they are accessed in a consistent order.
+
+.. _endhost_policy_selection:
+
+Exposing policies to the endhosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The path combinator finds the most recent FABRID map per AS among the received segments and subsequently uses this map to find the FABRID
+policies that are available for each interface pair of hops.
+This results in a set of *PolicyIdentifiers* per hop, which can then be used by the application, such as by defining an application parameter
```

Comment:
"This results in a set of *PolicyIdentifiers* per hop, which can then used by"

-> Suggestion to replace with: "This information can then be used by...

(the term PolicyIdentifiers, which is even highlighted in italic, seems not to be used anywhere else?)

Reply:
Suggestion:
"application parameter (we will use ``--fabridpolicy``), ..." (and remove "e.g. ``--fabridpolicy``" at the end of the sentence)